### ISSUE TITLE
OnlyEmailPasswordAuth not working

### ISSUE THREAD

**Issue by liron-navon** [AnthonyNahas/ngx-auth-firebaseui#177]
### Bug Report or Feature Request (mark with an `x`)
```
- [X ] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
OSX High Sierra

### Versions
```
Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.11.4
@angular-devkit/build-angular     0.13.0
@angular-devkit/build-optimizer   0.13.0
@angular-devkit/build-webpack     0.13.0
@angular-devkit/core              7.1.4
@angular-devkit/schematics        7.1.4
@angular/cdk                      7.3.0
@angular/fire                     5.1.1
@angular/flex-layout              7.0.0-beta.23
@angular/material                 7.3.0
@ngtools/webpack                  7.3.0
@schematics/angular               7.1.4
@schematics/update                0.11.4
rxjs                              6.3.3
typescript                        3.1.6
webpack                           4.29.0

```

### Repro steps

1. start a new cli project
2. follow the instructions
3. in the settings pass "onlyEmailPasswordAuth" as true.
4. The github/google/facebook... icons are still there.

### The log given by the failure
No error logged.

### Desired functionality
The Icons are not supposed to be seen?

**Comment by liron-navon** [comment]
@AnthonyNahas What are providers? the firebase/auth auth.Provider ?
Or just a list of strings?

**Comment by DannyHalm** [comment]
Hello Liron, 
Provider is Firebase auth providers, where you can decide which provider do you want to use as options for signin/signup in ngx-auth-firebaseui. Recently Anthony changed his code to allow us to show empty array as provider to limit the signing/singout to username password by steering provider=[] in your component. 
I hope it makes sense..best regards ..

**Comment by AnthonyNahas** [comment]
@Avrohom this has been fixed in [v2.3.0](https://www.npmjs.com/package/ngx-auth-firebaseui) which is live now
